production/helm/loki/CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ Entries should include a reference to the pull request that introduced the chang
 - [CHANGE] **BREKAING** Rename and change format of `enterprise.provisioner.tenants`. Property has been renamed to `enterprise.provisioner.additionalTenants`, and is now an array of objects rather than string. Each object must contain a `name` and a `secretNamespace` field, where `name` is the name of the tenant and `secretNamespace` is the namespace to create the secret with the tenant's read and write token.
 - [CHANGE] **BREAKING** Change the structure of `monitoring.selfMonitoring.tenant` from a string to an object. The new object must have a `name` and a `secretNamespace` field, where `name` is the name of the self-monitoring tenant and `secretNamespace` is the namespace to create an additional secret with the tenant's token. A secret will still also be created in the release namespace as it's needed by the Loki canary.
 - [CHANGE] **BREAKING** Remove ability to create self-monitoring resources in different namespaces (with the exception of dashboard configmaps).
+## 3.10.1
```

Comment:
In fact, looking at the proposed changes in `production/helm/loki/Chart.yaml` and `production/helm/loki/README.md`, I'm not sure if those align with the release process for the Loki HELM charts.

@trevorwhitney may have more context on this, but, shouldn't we have a _Next_ section here with the changes to be released with the next version of the charts? Otherwise, any new PR to these charts will result in a new chart version.

Reply:
Here's the conversation for the initial config https://github.com/grafana/loki/pull/7358

Let's not put it into a secret for now. We could add the option to do so though.

Reply:
Ooops :) Nice catch, sorry.

Fyi, in my company, we use https://keepachangelog.com/en/1.0.0/ . It recommends a `## [Unreleased]` section at the top of the changelog, comparable to the `Next`section you suggest.